playground/app/build.gradle.kts

```diff
@@ -19,7 +19,7 @@ android {
 
     buildTypes {
         release {
-            isMinifyEnabled = true
+            isMinifyEnabled = false
```

Comment:
What's the story here?

Reply:
During the Gradle build command, it tries to minify the release APK, which seems like an unnecessary waste of time.

Reply:
yeah